recipes/airss/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -ex
+
+LIBS="-llapack -lblas -lsymspg"
+
+make \
+  FC="$FC" \
+  FFLAGS="$FFLAGS" \
+  CC="$CC" \
+  CFLAGS="$CFLAGS" \
+  LDFLAGS="$LDFLAGS $LIBS" \
+  all
+
+mkdir -p "$PREFIX/bin"
+
+cp -v \
+  src/pp3/src/pp3 \
+  src/cabal/src/cabal \
+  src/buildcell/src/buildcell \
+  src/cryan/src/cryan \
+  external/symmol/symmol \
+  bin/* \
```

Comment:
I am not sure we should install
```
airss.pl*         casteps2res*      crud.pl*          gencell-091*      mc*               press*            repose_relax*     respack*          spawn*            vasp_relax*
airss_version*    castepsplit2res*  csymm*            genqe*            name*             prim*             rescat*           resplit*          spawn-slow*       xyzesplit2res*
ca*               check_airss*      despawn*          gulp_relax*       niggli*           psi4_relax*       resdel*           ress2xyz*         stopairss*        xyzs2res*
castep2res*       comp2minsep*      gap_relax*        lammps2cell*      pp0_relax*        python_relax*     reshake*          resview*          symm*
castep_relax*     conv*             gencell*          lammps_relax*     pp3_relax*        qe_relax*         resname*          run.pl*           tidy.pl*
```
Quite a few of these things sound rather generic and will certainly conflict with other packages in particular on the host environment.

I understand that you did not cook up these names, but maybe we should prefix all these with airss- or something like this?

Reply:
Prefixing will require some extensive patching in all the scripts. Would be nice if upstream would already provide properly prefixed names for all these scripts.

Reply:
That's going to be real difficult to automatically replace with `sed`...

```
❯ for bin in airss/bin/*; do bin=$(basename $bin) && echo "=== $bin ===" && grep --color=auto -rw $bin airss/bin airss/src/*/src/*.f90; done
=== airss.pl ===
airss/bin/check_airss:check_cmd airss.pl   "Add AIRSS/bin to your path"
airss/bin/check_airss:airss.pl -pp3 -max 2 -seed Al
airss/bin/check_airss:airss.pl -pp3 -cluster -max 2 -seed Al
airss/bin/check_airss:    airss.pl -gulp -max 2 -seed C2
airss/bin/check_airss:    airss.pl -press 100 -max 1 -step 0 -keep -seed C2
airss/bin/check_airss:	airss.pl -press 100 -mpinp $np -max 1 -step 0 -keep -seed C2
airss/bin/airss.pl:#                                  airss.pl                                        #
airss/bin/airss.pl:  printf STDERR "Usage: airss.pl [-pressure] [-prand] [-pmin] [-build] [-pp0] [-gosh] [-pp3] [-repose] [-gulp] [-lammps] [-gap] [-psi4] [-vasp] [-qe] [-python] [-castep] [-exec] [-cluster] [-slab] [-dos] [-workdir] [-max] [-num] [-amp] [-camp] [-mode] [-minmode] [-sim] [-symm] [-nosymm] [-mpinp] [-steps] [-best] [-track] [-keep] [-pack] [-harvest] [-seed]\n";
airss/bin/airss.pl:      if ($opt_build||$opt_pp0||$opt_gosh||$opt_pp3||$opt_gulp||$opt_lammps||$opt_gap||$opt_psi4||$opt_vasp||$opt_qe||$opt_python) {print "airss.pl: nbest not implemented\n";exit};
airss/bin/airss.pl:    system("sed -i 's/REM COMMAND_LINE/REM cmdline: airss.pl $commandline /g' $opt_workdir/$unique.res");
=== airss_version ===
airss/bin/pp3_relax:(printf " from code version ";airss_version)|sed 's/AIRSS Version//g' >> $seed.castep
airss/bin/repose_relax:(printf " from code version ";airss_version)|sed 's/AIRSS Version//g' >> $seed.castep
airss/bin/castep2res:airss_version=`airss_version`
airss/bin/castep2res:echo 'REM' $airss_version
=== ca ===
airss/bin/ca:    echo 'ca [-R(recursive)] [command line arguments for cryan]'
airss/bin/comp2minsep:name=`ca -l -de 0 -f $1 -r -t 1 | awk '{print $1}'` && [ ! -z "$name" ] && rescat $name | cryan -g 2>/dev/null | grep -E 'MINSEP|TARGVOL'
airss/bin/check_airss:ca -r
airss/bin/check_airss:ca -r
airss/bin/check_airss:    ca -r
airss/bin/check_airss:    ca -r
airss/bin/check_airss:	ca -r
=== castep2res ===
airss/bin/castepsplit2res:    castep2res ${f%.*}
airss/bin/run.pl:	system("castep2res $seed > $seed.res ; rm $seed-out.cell");
airss/bin/casteps2res:        castep2res ${f%.*} > ${f%.*}.res &
airss/bin/check_airss:check_cmd castep2res "Add AIRSS/bin to your path"
airss/bin/airss.pl:      system("castep2res $opt_cluster $opt_workdir/$unique > $opt_workdir/$unique.res");
airss/bin/airss.pl:      system("castep2res -1 $opt_workdir/$unique > $opt_workdir/$unique.res");
airss/bin/crud.pl:      system("castep2res $seed > $seed.res");
airss/bin/castep2res:#    echo 1>&2 "Usage: castep2res <cluster> <seed>"
airss/bin/castep2res:    echo 'Usage: castep2res [cluster] <seedname>'
=== castep_relax ===
airss/bin/airss.pl:	    $spe=`castep_relax -1 $executable $opt_sim $opt_symm $opt_workdir/$unique | awk '/Enthalpy:/ {print \$2}'`;
airss/bin/airss.pl:	  $spe=`castep_relax -1 $executable $opt_sim $opt_symm $opt_workdir/$unique | awk '/Enthalpy:/ {print \$2}'`;
airss/bin/airss.pl:    @results=`castep_relax $opt_steps $executable $opt_sim $opt_symm $opt_workdir/$unique`;
airss/bin/airss.pl:    @results=`castep_relax $opt_steps $executable $opt_sim $opt_symm $opt_workdir/$unique`;
airss/bin/airss.pl:    # Identify and remove repeated structure, keeping track of the number of encounters (not needed if castep_relax is used)
airss/bin/python_relax:    # taken from castep_relax
airss/bin/castep_relax:         echo 1>&2 "Usage: castep_relax <maxit> <exe> <sim> <symm> <seed>"
=== casteps2res ===
airss/bin/casteps2res:    echo 1>&2 "Usage: casteps2res"
=== castepsplit2res ===
airss/bin/castepsplit2res:    echo 1>&2 "Usage: castepsplit2res takes input from stdin"
airss/bin/airss.pl:    if ($opt_harvest) {system("mkdir -p $opt_workdir/harvest && castepsplit2res < $opt_workdir/$unique.castep > $opt_workdir/harvest/$unique.res");}
=== check_airss ===
=== comp2minsep ===
airss/bin/comp2minsep:    echo 'comp2minsep <comp>'
airss/src/buildcell/src/cell.f90:                        ctemp6='comp2minsep '//trim(composition)//' > '//trim(tempfile)
=== conv ===
airss/bin/qe_relax:    grep "enthalpy new" ${seed}.qe.out | awk '{print $4}' > ${seed}.conv
airss/bin/gulp_relax:    con="conv"  
airss/bin/mc:bc <<< $1*$2 | awk '{print "0",$1,"\n100",$1}' > ref.conv
airss/bin/conv:    echo 1>&2 "Usage: conv"
airss/bin/tidy.pl:unlink glob("*.conv");
airss/bin/airss.pl:    foreach (@delete_files) {if (($_ ne "$opt_workdir/$unique.res")&&($_ ne "$opt_workdir/$unique.conv")) {unlink($_);}}
airss/bin/airss.pl:  # Add this res file to the long list, and conv file if available
airss/bin/airss.pl:    if ( -e "$opt_workdir/$unique.conv" ) {
airss/bin/airss.pl:      system "cat $opt_workdir/$unique.conv >> $opt_workdir/$uniquenew.conv && rm -f $opt_workdir/$unique.conv"
airss/bin/castep_relax:    grep -q ": finished iteration" $seed.castep && grep ": finished iteration" $seed.castep | awk 'BEGIN {FS="enthalpy="};{print $2}' | awk '{print $1}' > $seed.conv
airss/bin/castep_relax:    echo >> $seed.conv
airss/src/buildcell/src/buildcell.f90:  logical :: conv=.false.,had_success=.false.
airss/src/buildcell/src/buildcell.f90:        call rash(conv,over)
airss/src/buildcell/src/buildcell.f90:        call opt_tpsd(conv,over)
airss/src/buildcell/src/buildcell.f90:     if(.not.conv)  write (stderr,'(a)') 'Did not converge'
airss/src/buildcell/src/buildcell.f90:     conv=.false.
airss/src/buildcell/src/buildcell.f90:     if(overlap+delta.gt.over/real(num_ions_total,dp)) conv=.true.
airss/src/buildcell/src/buildcell.f90:     if(.not.conv) then
airss/src/pp3/src/cell.f90:       write(convfile,'(a)') trim(seedname)//".conv"       
airss/src/pp3/src/cell.f90:998 stop 'There is a problem opening the conv file. Stopping.'
=== crud.pl ===
airss/bin/check_airss:check_cmd crud.pl    "Add AIRSS/bin to your path"
airss/bin/crud.pl:#                                   crud.pl                                        #
airss/bin/crud.pl:  printf STDERR "Usage: crud.pl [-mpinp xx] [-keep] [-nostop] [-cycle] Run many Castep computations\n";
airss/bin/crud.pl:      system("sed -i 's/REM COMMAND_LINE/REM cmdline: crud.pl $commandline /g' $seed.res");
=== csymm ===
airss/bin/symm:	cabal res xyz < $file > $file.xyz && csymm $file.xyz | head -1 | awk '{print "("$1")"}'
airss/bin/csymm:    echo 'csymm <xyz filename>'
airss/src/cryan/src/cryan.f90:                         ctemp="csymm "//trim(adjustl(unitfile))//" > "//trim(adjustl(unitfile))//".pg"
airss/src/cryan/src/cryan.f90:    character(len=7)  :: csymm
airss/src/cryan/src/cryan.f90:       csymm=trim(adjustl(structure(comp_index_inc(hull_index(n)))%symmetry))
airss/src/cryan/src/cryan.f90:            csymm,&
airss/src/cryan/src/cryan.f90:    character(len=7)  :: csymm
airss/src/cryan/src/cryan.f90:       csymm=trim(adjustl(structure(na)%symmetry))
airss/src/cryan/src/cryan.f90:            csymm,&
=== despawn ===
airss/bin/despawn:    echo 'despawn'
=== gap_relax ===
airss/bin/gap_relax:         echo 1>&2 "Usage: gap_relax <exe> <seed>"
airss/bin/airss.pl:    @results=`gap_relax ase_relax_cell.py $opt_workdir/$unique`;
=== gencell ===
airss/bin/gencell:  echo 'gencell <volume> <units> n x [<species> <number>]'
airss/bin/gencell-091:  echo 'gencell <volume> <units> n x [<species> <number>]'
=== gencell-091 ===
=== genqe ===
airss/bin/genqe:  echo 'genqe <volume> <units> n x [<species> <number>]'
=== gulp_relax ===
airss/bin/gulp_relax:         echo 1>&2 "Usage: gulp_relax <exe> <cluster> <pressure> <seed>"
airss/bin/airss.pl:      @results=`gulp_relax gulp 1 $opt_pressure $opt_workdir/$unique`;
airss/bin/airss.pl:      @results=`gulp_relax gulp 0 $opt_pressure $opt_workdir/$unique`;
=== lammps2cell ===
airss/bin/lammps_relax:lammps2cell $seed > $seed-out.cell
=== lammps_relax ===
airss/bin/airss.pl:    @results=`lammps_relax lammps $opt_pressure $opt_workdir/$unique`;
airss/bin/lammps_relax:         echo 1>&2 "Usage: lammps_relax <exe> <pressure> <seed>"
=== mc ===
airss/bin/mc:    echo 'mc <energy/fu> <nfu>'
=== name ===
airss/bin/qe_relax:	name=${seed##*/}
airss/bin/qe_relax:        known=`(find . -maxdepth 1 -name '*.res' | xargs cat; cabal cell res < $seed.cell | sed "1s/.*/TITL $name 0.0 0.0 0.0 0 0 $nat (P1) n - 1/" ) | cryan -c $sim $name 2> /dev/null | awk '{print $1}' | head -1`
airss/bin/resview:if find . -name "*.res" 2> /dev/null | xargs cat 2> /dev/null | grep -q "TITL $1" ;
airss/bin/ca:    find . -follow -name "*.res" 2>/dev/null | xargs cat 2>/dev/null | cryan $commandline
airss/bin/ca:    find . -maxdepth 1 -name "*.res" 2>/dev/null | xargs cat 2>/dev/null | cryan $commandline
airss/bin/comp2minsep:name=`ca -l -de 0 -f $1 -r -t 1 | awk '{print $1}'` && [ ! -z "$name" ] && rescat $name | cryan -g 2>/dev/null | grep -E 'MINSEP|TARGVOL'
airss/bin/airss.pl:	$known = `find . -name "*.res" | xargs cat | cryan -c $opt_sim $unique 2> /dev/null | awk '{print \$1}' | head -1`;
airss/bin/airss.pl:    my $best_struct=`(find . -maxdepth 1 -name "*.res" ; echo $opt_workdir/$unique.res) | xargs cat | cryan -r | grep "$best_formula" | head -1 | awk '{print \$1}'`;
airss/bin/rescat:	find . -name "*.res" 2> /dev/null | xargs cat 2> /dev/null | sed "/^TITL $var /,/~END/!d;/END/q"
airss/bin/genqe:echo Please modify ATOMIC_SPECIES in $seedname.qe to name your
airss/bin/name:  echo 'name <prefix>'
airss/bin/castep_relax:	name=${seed##*/}
airss/bin/castep_relax:        known=`(find . -maxdepth 1 -name '*.res' | xargs cat; cabal cell res < $seed.cell | sed "1s/.*/TITL $name 0.0 0.0 0.0 0 0 $nat (P1) n - 1/" ) | cryan -c $sim $name 2> /dev/null | awk '{print $1}' | head -1`
airss/src/buildcell/src/cell.f90:      ! * Check if the symmetry is defined by name
airss/src/buildcell/src/cell.f90:      character(len=16) :: name(100),cnum(100)
airss/src/buildcell/src/cell.f90:      name=''
airss/src/buildcell/src/cell.f90:         if(.not.any(name==ion_names(ni))) then
airss/src/buildcell/src/cell.f90:            name(nspec)=ion_names(ni)
airss/src/buildcell/src/cell.f90:         num(ns)=count(ion_names(1:num_ions).eq.name(ns))
airss/src/buildcell/src/cell.f90:      if(any(name(:).eq."C  ")) have_c=.true.
airss/src/buildcell/src/cell.f90:            if(name(ns).eq.elements_alpha(n)) exit
airss/src/buildcell/src/cell.f90:         if(name(ns).eq."O  ") then
airss/src/buildcell/src/cell.f90:         else if(name(ns).eq."C  ") then
airss/src/buildcell/src/cell.f90:         else if((name(ns).eq."H  ").and.have_c) then
airss/src/buildcell/src/cell.f90:           (trim(name(element_index(ns))),&
airss/src/cryan/src/cryan.f90:    character(len=10)  :: name(100)
airss/src/cryan/src/cryan.f90:          ! ** Remove numbers from ion name
airss/src/cryan/src/cryan.f90:       name=''
airss/src/cryan/src/cryan.f90:          if(.not.any(name==structure(ns)%ion_names(ni))) then
airss/src/cryan/src/cryan.f90:             name(structure(ns)%num_species)=structure(ns)%ion_names(ni)
airss/src/cryan/src/cryan.f90:       structure(ns)%species_names=name(1:structure(ns)%num_species)
airss/src/cryan/src/cryan.f90:             write (name(ni),'(i10)') structure(ns)%species_num(ni)
airss/src/cryan/src/cryan.f90:             name(ni)=''
airss/src/cryan/src/cryan.f90:            trim(adjustl(name(element_index(ni)))),ni=1,structure(ns)%num_species)
airss/src/cryan/src/cryan.f90:       print '(a)', '     find . -name "*.res" | xargs cat | cryan -m' 
airss/src/cryan/src/cryan.f90:       print '(a)', ' -fr, --formula_reference <formula>   Structure name for enthalpy reference' 
airss/src/pp3/src/pp3.f90:  subroutine get_arguments(name)
airss/src/pp3/src/pp3.f90:    character(len=80), intent(out) :: name
airss/src/pp3/src/pp3.f90:    name = trim(cargs(num_args))
=== niggli ===
airss/bin/niggli:    echo 1>&2 "Usage: niggli"
airss/bin/niggli:   cabal res res 0 < $f > $f.niggli && mv $f.niggli $f
airss/src/cabal/src/cabal.f90:        call niggli() ! ** Replace with spglib
airss/src/cabal/src/cabal.f90:  subroutine niggli()
airss/src/cabal/src/cabal.f90:  end subroutine niggli
=== pp0_relax ===
airss/bin/pp0_relax:         echo 1>&2 "Usage: pp0_relax <exe> <seed>"
airss/bin/airss.pl:    @results=`pp0_relax pp0 $opt_workdir/$unique`;
airss/bin/airss.pl:    @results=`pp0_relax gosh $opt_workdir/$unique`;
=== pp3_relax ===
airss/bin/airss.pl:    @results=`pp3_relax pp3 $opt_workdir/$unique`;
=== press ===
airss/bin/qe_relax:         echo 1>&2 "Usage: qe_relax <maxit> <exe> <press [GPa]> <sim> <symm> <seed>"
airss/bin/qe_relax:    # If 'press' is already present, modify it
airss/bin/qe_relax:    if grep -q -w press ${seed}.qe
airss/bin/qe_relax:       sed -i "/\<press\>/c\  press = $( echo 10*${pressure} | bc -l )D0 ," ${seed}.qe
airss/bin/qe_relax:       sed -i "s/&CELL/&\n  press = $( echo 10*${pressure} | bc -l )D0 ,/" ${seed}.qe
airss/bin/qe_relax:       echo "  press = "$( echo 10*${pressure} | bc -l )"D0 ," >> ${seed}.qe
airss/bin/gulp_relax:press=$3
airss/bin/gulp_relax:echo "pressure $press GPa" >> $seed.gin
airss/bin/press:  echo 'press <p_min> <p_step> <p_num> <seed>'
airss/bin/check_airss:    airss.pl -press 100 -max 1 -step 0 -keep -seed C2
airss/bin/check_airss:	airss.pl -press 100 -mpinp $np -max 1 -step 0 -keep -seed C2
airss/bin/lammps_relax:variable p equal press*0.000101325               # convert atm back to GPa
airss/bin/genqe:echo "  press = xxx , " >> $seedname.qe
airss/src/cryan/src/cryan.f90:  real(kind=dp)                                   :: dmod=1.0_dp,alpha=-1.0_dp,vscale,lstsq(2),pmin,pmax,press
airss/src/cryan/src/cryan.f90:             press=pmin+(pmax-pmin)*real(n-1,dp)/real(num_interp-1,dp)
airss/src/cryan/src/cryan.f90:             peinterp(n,1,na)=press
airss/src/cryan/src/cryan.f90:             peinterp(n,2,na)=gp_predict(press,1)
=== prim ===
airss/bin/prim:    echo 1>&2 "Usage: prim"
=== psi4_relax ===
airss/bin/psi4_relax:         echo 1>&2 "Usage: psi4_relax <exe> <seed>"
airss/bin/airss.pl:    @results=`psi4_relax psi4 $opt_workdir/$unique`;
=== python_relax ===
airss/bin/airss.pl:    @results=`python_relax "python relax.py" $opt_pressure $opt_workdir/$unique`;
airss/bin/python_relax:         echo 1>&2 "Usage: python_relax <exe> <pressure> <seed>"
=== qe_relax ===
airss/bin/qe_relax:         echo 1>&2 "Usage: qe_relax <maxit> <exe> <press [GPa]> <sim> <symm> <seed>"
airss/bin/airss.pl:    @results=`qe_relax $opt_steps $executable $opt_pressure $opt_sim $opt_symm $opt_workdir/$unique`;
airss/bin/genqe:# qe_relax fills in fields labelled 'xxx'
=== repose_relax ===
airss/bin/repose_relax:         echo 1>&2 "Usage: repose_relax <exe> <nomp> <seed>"
airss/bin/airss.pl:	  $spe=`(repose_relax 'repose -p $opt_pressure' $opt_mpinp $opt_workdir/$unique) | awk '/Enthalpy:/ {print \$2}'`;
airss/bin/airss.pl:    @results=`(repose_relax repose $opt_mpinp $opt_workdir/$unique)`;
airss/bin/crud.pl:	system("(repose_relax repose $opt_mpinp $seed)")
airss/bin/crud.pl:	system("repose_relax repose 1 $seed")
=== rescat ===
airss/bin/resview:    rescat $1 | cabal res res 0.1 | cabal res res 0
airss/bin/comp2minsep:name=`ca -l -de 0 -f $1 -r -t 1 | awk '{print $1}'` && [ ! -z "$name" ] && rescat $name | cryan -g 2>/dev/null | grep -E 'MINSEP|TARGVOL'
airss/bin/rescat:    echo 1>&2 "Usage: rescat <seeds>"
=== resdel ===
airss/bin/resdel:    echo 1>&2 "Usage: resdel <seeds>"
=== reshake ===
airss/bin/reshake:    echo 1>&2 "Usage: reshake  <posamp> <cellamp> <maxatoms> <nstructures> <seed>"
=== resname ===
airss/bin/resname:    echo 1>&2 "Usage: resname"
=== respack ===
airss/bin/respack:    echo 1>&2 "Usage: respack <seed>"
=== resplit ===
airss/bin/resplit:    echo 1>&2 "Usage: resplit takes input from stdin"
airss/bin/resplit:tmpfile=$(mktemp -u /tmp/resplit.XXXXXXXXX-)
=== ress2xyz ===
airss/bin/ress2xyz:    echo 1>&2 "Usage: ress2xyz"
=== resview ===
airss/bin/resview:    echo 1>&2 "Usage: resview <seed>"
=== run.pl ===
airss/bin/run.pl:#                                   run.pl                                         #
airss/bin/run.pl:  printf STDERR "Usage: run.pl [-mpinp] [-conventional] [-keep] [-check] Run many Castep computations\n";
airss/bin/run.pl:	system("sed -i 's/REM COMMAND_LINE/REM cmdline: run.pl $commandline /g' $seed.res");
airss/bin/check_airss:check_cmd run.pl     "Add AIRSS/bin to your path"
=== spawn ===
airss/bin/stopairss:for i in `grep -v \# ~/.spawn | awk '{print $1}'`
airss/bin/spawn-slow:#                                   spawn-slow                                     #
airss/bin/spawn-slow:    echo 'spawn [command]'
airss/bin/spawn-slow:for i in `grep -v \# ~/.spawn | awk 'BEGIN { FS = "slots=" } ; {print $1}'`
airss/bin/spawn-slow:    slots=`awk -v host=$i '$1==host {print $2}' ~/.spawn | awk 'BEGIN { FS = "slots=" } ; {print $2}'`
airss/bin/spawn-slow:    root=`awk -v host=$i '$1==host {print $3}' ~/.spawn | awk 'BEGIN { FS = "root=" } ; {print $2}'`
airss/bin/check_airss:if [[ -f ~/.spawn ]]; then
airss/bin/check_airss:    echo '.spawn +'
airss/bin/check_airss:    echo '.spawn - '
airss/bin/spawn:#                                      spawn                                       #
airss/bin/spawn:    echo 'spawn [command]'
airss/bin/spawn:for i in `grep -v \# ~/.spawn | awk 'BEGIN { FS = "slots=" } ; {print $1}'`
airss/bin/spawn:    (slots=`awk -v host=$i '$1==host {print $2}' ~/.spawn | awk 'BEGIN { FS = "slots=" } ; {print $2}'`
airss/bin/spawn:    root=`awk -v host=$i '$1==host {print $3}' ~/.spawn | awk 'BEGIN { FS = "root=" } ; {print $2}'`
=== spawn-slow ===
airss/bin/spawn-slow:#                                   spawn-slow                                     #
=== stopairss ===
airss/bin/check_airss:# Issue the stopairss command on ^C
=== symm ===
airss/bin/qe_relax:         echo 1>&2 "Usage: qe_relax <maxit> <exe> <press [GPa]> <sim> <symm> <seed>"
airss/bin/qe_relax:symm=$5
airss/bin/qe_relax:    if [[ $symm -gt 0 ]]; then
airss/bin/qe_relax:	cabal cell cell -$symm | cabal cell cell 0 > $seed-out.cell.temp && mv $seed-out.cell.temp $seed-out.cell
airss/bin/symm:    echo 'symm [-cl] <filename>'
airss/bin/airss.pl:  printf STDERR "Usage: airss.pl [-pressure] [-prand] [-pmin] [-build] [-pp0] [-gosh] [-pp3] [-repose] [-gulp] [-lammps] [-gap] [-psi4] [-vasp] [-qe] [-python] [-castep] [-exec] [-cluster] [-slab] [-dos] [-workdir] [-max] [-num] [-amp] [-camp] [-mode] [-minmode] [-sim] [-symm] [-nosymm] [-mpinp] [-steps] [-best] [-track] [-keep] [-pack] [-harvest] [-seed]\n";
airss/bin/airss.pl:  printf STDERR "       -symm     f  Symmetrise on-the-fly (0.0)\n";
airss/bin/airss.pl:            "symm=f"     => \$opt_symm,
airss/bin/castep2res:    sym=`symm -cl $seed.res.temp`
airss/bin/castep2res:    sym=`symm $seed.res.temp`
airss/bin/castep_relax:         echo 1>&2 "Usage: castep_relax <maxit> <exe> <sim> <symm> <seed>"
airss/bin/castep_relax:symm=$4
airss/bin/castep_relax:    if [[ $symm != 0 ]]; then
airss/bin/castep_relax:	cabal cell cell -$symm < $seed-out.cell | cabal cell cell 0 > $seed-out.cell~ && mv $seed-out.cell~ $seed-out.cell
=== tidy.pl ===
airss/bin/tidy.pl:   printf STDERR "Usage: tidy.pl Tidy up the result of a AIRSS run\n";
=== vasp_relax ===
airss/bin/vasp_relax:         echo 1>&2 "Usage: vasp_relax <exe> <seed>"
airss/bin/airss.pl:    @results=`vasp_relax $executable $opt_workdir/$unique`;
=== xyzesplit2res ===
airss/bin/xyzesplit2res:    echo 1>&2 "Usage: xyzesplit2res takes input from stdin"
=== xyzs2res ===
airss/bin/xyzs2res:    echo 1>&2 "Usage: xyzs2res [padding]"
```

Reply:
Fixed now, but also providing a package ``airss-with-default-names`` to opt-in for having all these script pollute the host environment.